language/de/ucp.php

```diff
@@ -73,22 +73,26 @@
+		<li>Weiterhin werden die Daten gespeichert, die du bei der Registrierung, in deinem Profil oder deinem persönlichem Bereich angibst. Für die Registrierung sind mindestens ein eindeutiger Benutzername, eine E-Mail-Adresse und ein Passwort notwendig. Wenn durch den Betreiber weitere Daten als notwendig festgelegt wurden, so ist dies für dich vor deren Eingabe ersichtlich.</li>
+		<li>Wenn du einen Beitrag oder eine private Nachricht erstellst, so werden die dort eingegebenen Daten ebenfalls gespeichert. Gleiches gilt, wenn du einen Beitrag als Entwurf zwischenspeicherst. In diesen Fällen wird auch deine IP-Adresse gespeichert. Die IP-Adresse wird weiterhin bei folgenden Aktionen gespeichert: Löschen und Ändern von Beiträgen (dazu zählen Private Nachrichten und Umfragen), Änderungen an zentralen Profildaten (E-Mail-Adresse, Kontoaktivierung, Benutzer-Passwort) und gescheiterte Anmeldeversuche. Die von deinem Browser übermittelte Browser-Kennzeichnung (User Agent) wird nur in der „Wer ist online?“-Funktion angezeigt und nicht zwischengespeichert.</li>
+		<li>Schließlich erfordern einzelne Funktionen des Boards, dass weitere Daten gespeichert werden. Dazu gehören dein Abstimmungsverhalten bei Umfragen, der Gelesen-Status von deinen Beiträgen oder explizit von dir gesetzte Lesezeichen oder Benachrichtigungsfunktionen.</li></ol>
 		<p>Dein Passwort wird mit einer Einwege-Verschlüsselung (Hash) gespeichert, so dass es sicher ist. Jedoch wird dir empfohlen, dieses Passwort nicht auf einer Vielzahl von Webseiten zu verwenden. Das Passwort ist dein Schlüssel zu deinem Benutzerkonto für das Board, also geh mit ihm sorgsam um. Insbesondere wird dich kein Vertreter des Betreibers, von phpBB Limited oder ein Dritter berechtigterweise nach deinem Passwort fragen. Solltest du dein Passwort vergessen haben, so kannst du die Funktion „Ich habe mein Passwort vergessen“ benutzen. Die phpBB-Software fragt dich dann nach deinem Benutzernamen und deiner E-Mail-Adresse und sendet anschließend ein neu generiertes Passwort an diese Adresse, mit dem du dann auf das Board zugreifen kannst.</p>
 
 		<h3>Gestattung der Datenspeicherung</h3>
-		<p>Du gestattest dem Betreiber, die von dir im Rahmen der Registrierung eingegebenen Daten sowie laufende Zugriffsdaten (Datum und Uhrzeit der Nutzung, IP-Adresse und weitere von deinem Browser übermittelte Daten) zu speichern und für den Betrieb des Boards zu verwenden.</p>
+		<p>Du gestattest dem Betreiber, die von dir eingegebenen und oben näher spezifizierten Daten zu speichern, um das Board betreiben und anbieten zu können.</p>
+
+		<p>Darüber hinaus ist der Betreiber berechtigt im Rahmen einer Interessenabwägung zwischen deinen und seinen Interessen sowie den Interessen Dritter, Zeitpunkte von Zugriffen und Aktionen zusammen mit deiner IP-Adresse und der von deinem Browser übermittelter Browser-Kennung zu speichern, sofern dies zur Gefahrenabwehr oder zur rechtlichen Nachverfolgbarkeit von Äußerungen notwendig ist.</p>
```

Comment:
```suggestion
		<p>Darüber hinaus ist der Betreiber berechtigt, im Rahmen einer Interessenabwägung zwischen deinen und seinen Interessen sowie den Interessen Dritter, Zeitpunkte von Zugriffen und Aktionen zusammen mit deiner IP-Adresse und der von deinem Browser übermittelter Browser-Kennung zu speichern, sofern dies zur Gefahrenabwehr oder zur rechtlichen Nachverfolgbarkeit notwendig ist.</p>
```

Es geht ja um die Nachverfolgbarkeit von Änderungen etc.